test/test_helper.exs

```diff
@@ -1 +1 @@
-ExUnit.start()
+ExUnit.start(capture_log: true)
```

Comment:
This change seems unrelated to the name change.

Reply:
Derp. Forgot that was in there. This was me not wanting to see all the log messages when running the tests. I'll break it out

Reply:
Fixed